assets/sass/components/global/_googlesitekit-entity-search.scss

```diff
@@ -40,9 +39,16 @@
 		}
 
 		.googlesitekit-entity-search__close {
+			border-radius: 50%;
 			height: 32px;
-			min-width: 36px;
+			min-height: 36px;
+			min-width: 36px !important;
 			padding: 0;
+
+			@media ( min-width: $bp-wpAdminBarTablet ) {
```

Comment:
@hussain-t Can you check the breakpoints closely. I see sometime we are using `$bp-tablet` and sometimes `$bp-wpAdminBarTablet` 👍